src/getTarget.py

```diff
@@ -330,6 +332,9 @@ def resolveTarget(y, x, z, azimuth, theta, elevationData, xParams, yParams):
     finalDist = sqrt(finalHorizDist ** 2 + finalVertDist ** 2)
     terrainAlt = parseGeoTIFF.getAltFromLatLon(curY, curX, xParams, yParams, elevationData)
 
+    # send CoT message after resolving target location
+    create_and_send_cot(curX, curY, curZ, finalDist)
```

Comment:
See above comments for a revised set of defaults, including ce based on ray slant angle